app/views/users/_contests.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container outer-home-container col-md-5">
+    <div class="display-container no-margin-top">
+        <div class="row">
+            <h1 class="headings info-text col-xs-10 col-xs-offset-1 left">My Contests</h1>
+            <div class="info-text col-xs-1 plus-div right" data-toggle="tooltip" data-placement="bottom" title="Add Contest">
+                <%= link_to(new_contest_path, method: :get) do %>
+                    <%= content_tag('i', nil, class: 'glyphicon glyphicon-plus gly-plus') %>
+                <% end %>
+            </div>
+        </div>
+        <div class="row">
+            <%= form_tag user_path(@user), method: 'get', id: 'contest_form', class: 'form-inline' do %>
+                <%= text_field_tag :contest_search, params[:contest_search], class: 'form-control' %>
+                <%= submit_tag "Search", description: nil, class: 'btn btn-danger' %>
+            <% end %>
+        </div>
+        <% if @contests.count > 0 %>
+            <% if @contests.count > @per_page %>
+                <p><%= "#{@contests.count} found (displaying #{getRangeFromPage(params[:contest_page], @contests.length)})" %></p>
+            <% else %>
+                <p><%= "#{@contests.count} found" %></p>
+            <% end %>
+        <% else %>
+            <p>No contests found</p>
+        <% end %>
+        <div class="row">
```

Comment:
This `if` statement has appeared three times in this change:  matches, tournaments, and contests.  I think it should be abstracted into a helper method that looks something like this untested code:
```
def display_collection_count(collection, limit)
  name = collection.class.to_s
  if collection.count > 0
    text = "<p>#{collection.count} found"
    if collection.count > limit
      text += "(displaying #{getRangeFromPage(params[(name + '_page').to_sym], collection.length)})"
    end
    text += "</p>"
  else
    "<p>No #{name}s found</p>"
  end
end
```
and calling it something like `display_collection_count(@contests)`